app/controllers/smarty_streets_controller.rb

```diff
@@ -9,10 +9,13 @@ def street_address
     render json: get_data_on_address_zip(params), status: 200
   end
 
-  # This endpoint appears unused
-  def suggest
-    render json: get_suggestions_for_address(params), status: 200
-  end
+  # This endpoint appears unused (TheNotary 1/16)
+  # Lets remove it and see if anything breaks! (jparr 4/23)
+  # todo: remove if not used
+
+  # def suggest
+  #   render json: get_suggestions_for_address(params), status: 200
+  # end
```

Comment:
action already had a comment that it was unused from 2016 so I figured why not do this? Up to you if you think this is a good idea though.